unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEditorHUDController.cs

```diff
@@ -11,7 +11,7 @@ public class BackpackEditorHUDController
         private const string HAIR_CATEGORY = "hair";
         private const string EYEBROWS_CATEGORY = "eyebrows";
         private const string FACIAL_HAIR_CATEGORY = "facial_hair";
-        private const string BODYSHAPE_CATEGORY = "bodyshape";
+        private const string BODYSHAPE_CATEGORY = "body_shape";
```

Comment:
Isnt this going to break plenty of things? AFAIK the category has always been "bodyshape"

Reply:
No, we're only using this id to manage the new slots panel (each slot represents a wearable's category) and it's not an id that comes from outside.

![image](https://user-images.githubusercontent.com/64659061/237049146-36e7265e-b732-4a82-b03a-1738ecaa7906.png)